model/qti/container/Container.php

```diff
@@ -22,6 +22,7 @@
 
 namespace oat\taoQtiItem\model\qti\container;
 
+use Monolog\Logger;
```

Comment:
did not notice that in the initial PR, 
asking now, what is the purpose of adding it here?